src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
 
-        if (values.requestorAddressZipCode && !ValidationUtils.isValidZipCode(values.requestorAddressZipCode)) {
-            errors.requestorAddressZipCode = 'bankAccount.error.zipCode';
+    if (values.dob) {
+        if (!ValidationUtils.isValidPastDate(values.dob) || !ValidationUtils.meetsMaximumAgeRequirement(values.dob)) {
+            errors.dob = 'bankAccount.error.dob';
+        } else if (!ValidationUtils.meetsMinimumAgeRequirement(values.dob)) {
+            errors.dob = 'bankAccount.error.age';
         }
+    }
 
-        if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
-            errors.isControllingOfficer = 'requestorStep.isControllingOfficerError';
-        }
+    if (values.ssnLast4 && !ValidationUtils.isValidSSNLastFour(values.ssnLast4)) {
+        errors.ssnLast4 = 'bankAccount.error.ssnLast4';
+    }
 
-        return errors;
+    if (values.requestorAddressStreet && !ValidationUtils.isValidAddress(values.requestorAddressStreet)) {
+        errors.requestorAddressStreet = 'bankAccount.error.addressStreet';
     }
 
-    submit(values) {
-        const payload = {
-            bankAccountID: lodashGet(this.props.reimbursementAccount, 'achData.bankAccountID') || 0,
-            ...values,
-        };
+    if (values.requestorAddressZipCode && !ValidationUtils.isValidZipCode(values.requestorAddressZipCode)) {
+        errors.requestorAddressZipCode = 'bankAccount.error.zipCode';
+    }
 
-        BankAccounts.updatePersonalInformationForBankAccount(payload);
+    if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
+        errors.isControllingOfficer = 'requestorStep.isControllingOfficerError';
     }
 
-    render() {
-        if (this.props.shouldShowOnfido) {
-            return (
-                <RequestorOnfidoStep
-                    reimbursementAccount={this.props.reimbursementAccount}
-                    reimbursementAccountDraft={this.props.reimbursementAccountDraft}
-                    onBackButtonPress={this.props.onBackButtonPress}
-                    getDefaultStateForField={this.props.getDefaultStateForField}
-                />
-            );
-        }
+    return errors;
+};
 
+function InnerRequestorStep({reimbursementAccount, shouldShowOnfido, reimbursementAccountDraft, onBackButtonPress, getDefaultStateForField, translate}, ref) {
+    const defaultValues = useMemo(
+        () => ({
+            firstName: getDefaultStateForField('firstName'),
+            lastName: getDefaultStateForField('lastName'),
+            street: getDefaultStateForField('requestorAddressStreet'),
+            city: getDefaultStateForField('requestorAddressCity'),
+            state: getDefaultStateForField('requestorAddressState'),
+            zipCode: getDefaultStateForField('requestorAddressZipCode'),
+            dob: getDefaultStateForField('dob'),
+            ssnLast4: getDefaultStateForField('ssnLast4'),
```

Comment:
Can we re-use `INPUT_KEYS`?
```suggestion
            firstName: getDefaultStateForField(INPUT_KEYS.firstName),
            lastName: getDefaultStateForField(INPUT_KEYS.lastName),
            street: getDefaultStateForField(INPUT_KEYS.street),
            ....
```

Reply:
Done ✅